configs/patternfly.json

```diff
@@ -13,30 +13,317 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "header nav [aria-current]",
+      "selector": "header nav li a",
       "global": true,
       "default_value": "Documentation"
     },
     "lvl1": {
-      "selector": "#page-sidebar [aria-expanded='true']",
+      "selector": "#page-sidebar li a",
       "global": true,
       "default_value": "Chapter"
     },
-    "lvl2": "main h1",
-    "lvl3": "main h2",
-    "lvl4": "main h3",
-    "lvl5": "main h4",
-    "lvl6": "main h5",
+    "lvl2": {
+      "selector": "#page-sidebar section li a",
+      "global": true,
+      "default_value": "Chapter"
+    },
+    "lvl3": "main h1",
+    "lvl4": "main h2",
+    "lvl5": "main h3",
+    "lvl6": "main h4",
+    "lvl7": "main h5",
     "text": "main p, main li, main td"
   },
   "selectors_exclude": [
+    ".pf4-l-footer",
     "[class*='live-preview']"
   ],
-  "custom_settings": {
-    "separatorsToIndex": "_"
-  },
   "conversation_id": [
     "772189694"
   ],
-  "nb_hits": 15754
+  "nb_hits": 15754,
+  "custom_settings": {
+    "separatorsToIndex": "_",
+    "synonyms": [
+      [
+        "about modal",
```

Comment:
The scraper supports only supports regular *one* word synonyms. Please remove the 2 words one.

Reply:
Okay, in the examples I saw something like `ECMAScript 6`, so that is also not valid, right?

Reply:
Good point, will fix it. Thanks for pointing out.